src/main/java/mate/academy/dao/impl/MovieDaoImpl.java

```diff
@@ -45,6 +46,12 @@ public Optional<Movie> get(Long id) {
 
     @Override
     public List<Movie> getAll() {
-        return null;
+        try (Session session = HibernateUtil.getSessionFactory().openSession()) {
+            Query<Movie> getAllFromMoviesQuery
```

Comment:
The variable name `getAllFromMoviesQuery` is somewhat verbose. Consider simplifying it to `getAllMoviesQuery` for clarity and consistency with typical naming conventions.